packages/swirl-components/src/components/swirl-shell-navigation-item/swirl-shell-navigation-item.tsx

```diff
@@ -17,15 +19,25 @@ export class SwirlShellNavigationItem {
   @Prop() boxed?: boolean;
   @Prop() hideLabel?: boolean;
```

Comment:
Issue: Not related to your changes, but I noticed that the tooltip is displayed when "hideLabel" is undefined. Could you change the default value to `false`. Tis should fix the issue.

Reply:
yeah you're right, i changed it